src/lib/keypom.ts

```diff
@@ -190,7 +207,21 @@ class KeypomJS {
     return null;
   };
 
-  getDrops = async ({ accountId, start, limit }) => await getDrops({ accountId, start, limit });
+  getDrops = async ({ accountId, start, limit }) => {
+    /** Get Drops caching logic */
+    if (
+      Date.now() > this.dropStore.getDropsExpiryTime ||
+      start !== this.dropStore.getDropsLastPage
```

Comment:
i tested locally, if I paginate to next page (page 1 -> 2), then back to previous (2 -> 1), it's not showing the previous drops from store (1)